pkg/debugapi/chequebook.go

```diff
@@ -235,6 +235,14 @@ func (s *Service) swapCashoutHandler(w http.ResponseWriter, r *http.Request) {
 		ctx = sctx.SetGasLimit(ctx, l)
 	}
 
+	if !s.cashOutChequeSem.TryAcquire(1) {
+		s.logger.Debug("debug api: cashout resource is busy")
+		jsonhttp.InternalServerError(w, "cashout resource is busy")
```

Comment:
for the message logged in the terminal I would add something like "simultaneous on-chain operations not supported" . for the error code I would consider to return either `HTTP 503 Service Unavailable` or `HTTP 429 Too many requests`